<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWindow</class>
 <widget class="QDialog" name="InfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTiles">
      <attribute name="title">
       <string>Tiles</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Video</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QLabel" name="labelVideoMode">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Mode (BGMODE)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>BG1</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="3" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Tilemap (BG1SC)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="labelBG1Tilemap">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="labelBG1Tileset">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>H Offset (BG1HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="labelBG1ChrSize">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Enabled (TM, TS)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Chr Size (BGMODE)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="labelBG1HOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelBG1Enabled">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Tileset (BG12NBA)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_41">
              <property name="text">
               <string>V Offset (BG1HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="labelBG1VOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>BG2</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Chr Size (BGMODE)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Tileset (BG12NBA)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelBG2Enabled">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="labelBG2HOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_43">
              <property name="text">
               <string>H Offset (BG2HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="labelBG2ChrSize">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>Enabled (TM, TS)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Tilemap (BG2SC)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="labelBG2Tilemap">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="labelBG2Tileset">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_45">
              <property name="text">
               <string>V Offset (BG2HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="labelBG2VOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>BG3</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="4" column="1">
             <widget class="QLabel" name="labelBG3HOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelBG3Enabled">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="labelBG3ChrSize">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="labelBG3Tilemap">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Tileset (BG34NBA)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="labelBG3Tileset">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>Chr Size (BGMODE)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Tilemap (BG3SC)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>Enabled (TM, TS)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_47">
              <property name="text">
               <string>H Offset (BG3HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_49">
              <property name="text">
               <string>V Offset (BG3HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="labelBG3VOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>BG4</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="3" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Tilemap (BG4SC)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_51">
              <property name="text">
               <string>H Offset (BG4HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Tileset (BG34NBA)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelBG4Enabled">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="labelBG4ChrSize">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="labelBG4HOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="labelBG4Tileset">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_38">
              <property name="text">
               <string>Enabled (TM, TS)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Chr Size (BGMODE)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="labelBG4Tilemap">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_53">
              <property name="text">
               <string>V Offset (BG4HOFS)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="labelBG4VOFS">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tileset</string>
          </attribute>
          <widget class="QGraphicsView" name="gvTiles">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>431</width>
             <height>541</height>
            </rect>
           </property>
          </widget>
          <widget class="QGraphicsView" name="gvPalette">
           <property name="geometry">
            <rect>
             <x>470</x>
             <y>10</y>
             <width>262</width>
             <height>262</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tilemap</string>
          </attribute>
          <widget class="QGraphicsView" name="gvTilemap">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>520</width>
             <height>520</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMemory">
      <attribute name="title">
       <string>Memory</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>291</width>
         <height>381</height>
        </rect>
       </property>
       <property name="title">
        <string>Cartridge</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="17" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Game Code</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="labelRomSize">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelChipsetSubtype">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="labelRomVersion">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelTitle">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="labelRamSize">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>ROM Version</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="labelExpansionRamSize">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLabel" name="labelMakerCode">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Maker Code</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="labelExpansionFlashSize">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Dev Id</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="labelSpecialVersion">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Checksum 2</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="labelChecksum1">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>ROM Type</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Expansion RAM Size</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Checksum 1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ROM Speed</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="labelChecksum2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Expansion Flash Size</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Special Version</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ROM Size</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>RAM Size</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelRomType">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelChipset">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Chipset</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="labelCountry">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Ttile</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Country</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="labelGameCode">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelRomSpeed">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="labelDevId">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Chipset Subtype</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Interleaved</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="labelInterleaved">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSprites">
      <attribute name="title">
       <string>Sprites</string>
      </attribute>
      <widget class="QGraphicsView" name="gvObjTable1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>260</y>
         <width>431</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvObjTable2">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>260</y>
         <width>431</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>221</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Sprite Info</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="1">
         <widget class="QLabel" name="labelObjTable1Addr">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Table 1 Address</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>Table 2 Address</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelObjTable2Addr">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>Sprite Sizes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelObjSizes">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
